File: Práctica/2.5 - RobocodeLabo II/src/CasseraBottaBianchi/Estrategia.java
package CasseraBottaBianchi;

public interface Estrategia {
    public void run(CasseraBottaBianchiRobot robot);
    public void onScannedRobot(CasseraBottaBianchiRobot robot);
    public void onHitByBullet(CasseraBottaBianchiRobot robot);
    public void onHitWall(CasseraBottaBianchiRobot robot);
}

File: Talleres/4/src/Estrategia.kt
package CasseraBottaBianchi

interface Estrategia {
    fun run(robot: LaboRobot)
    fun onScannedRobot(robot: LaboRobot)
    fun onHitByBullet(robot: LaboRobot)
    fun onHitWall(robot: LaboRobot)
}
